helm/coredns-app/templates/pdb.yaml

```diff
@@ -0,0 +1,12 @@
+{{- if (gt (.Values.maxReplicas | int) 1) }}
+apiVersion: policy/v1beta1
+kind: PodDisruptionBudget
+metadata:
+  name: {{ .Values.name }}
+  namespace: {{ .Values.namespace }}
```

Comment:
Do we need labels here?

Seems like some templates have them but others don't.

Reply:
The namespace? I think it's okay.

Reply:
I think Ross meant gs labels as other resources have

Reply:
3 other GS labels from below,
```
  labels:
    giantswarm.io/service-type: "{{ .Values.serviceType }}"
    k8s-app: {{ .Values.name }}
    kubernetes.io/name: "CoreDNS"
```

Not sure we need those values for `pdb`.

Reply:
I will hold to release this until we make it sure not needed, sorry.

Reply:
I think all the resources should have these labels and that's what we say in `fmt`.

https://github.com/giantswarm/fmt/blob/master/kubernetes/annotations_and_labels.md#common-labels

Ji, can you add them and then create the release?

Reply:
Here the PR https://github.com/giantswarm/coredns-app/pull/18